Implemented some functionality to the guess command

JSteve0 opened the pull request:
Added guess command to cli
Created guess method with most functionality
Added tests

relates to #16

mharrison7787 commented on src/app.js:
```diff
@@ -30,4 +33,11 @@ vorpal
     callback();
   });
 
+vorpal
```
Even if its self explanatory I think we should add comments anyway so we get good grades.

JSteve0 replied:
Added more documentation.

mharrison7787 commented on src/app.js:
```diff
@@ -6,6 +6,9 @@ var vorpal = require('vorpal')();
 //GameManager object
 var GameManager = new (require('./classes/GameManager.js'))();
 
```
Go more detail of what the variable is doing. I know its self explanatory but just in case.

JSteve0 replied:
Added more detail and resolved.

mharrison7787 reviewed (changes requested):
I added some inline comments that you should address before merging. Just so the whole group can understand the code

JSteve0 commented:
> I added some inline comments that you should address before merging. Just so the whole group can understand the code

Fixed those issues with a new commit, thanks for the feedback!

mharrison7787 reviewed (approved):
Looks great now that each definition has at least one comment. Thanks for adding these.

mharrison7787 commented:
Thanks for fixing this. Looks good now with the comments in each file
